llvm-external-projects/iree-dialects/include/iree-dialects/Dialect/LinalgExt/Passes/Passes.h

```diff
@@ -120,15 +120,17 @@ std::unique_ptr<Pass> createConvertConv2DToWinogradPass();
 // Transform dialect version of tile and decompose attention wrapper.
 void tileAndDecomposeAttention(IREE::LinalgExt::AttentionOp attnOp,
                                SmallVectorImpl<Operation *> &ops,
-                               RewriterBase &rewriter, bool onlyTile = false);
+                               RewriterBase &rewriter, bool onlyTile = false,
+                               uint64_t tileSize = 0);
```

Comment:
Can you add a comment on what this parameter does? Also, I prefer Optional instead of using 0 as the default value. Especially since tileSize is something you might divide by.

Reply:
Agreed, optional would be nicer. I will add a comment as well.